tests/unit/projection-test.js

```diff
@@ -157,120 +155,119 @@ module('unit/projection', function(hooks) {
           the FETCHED_PROJECTION_ID is the unfetched base-record
           the UNFETCHED_PROJECTION_ID is the already fetched base-record
         */
-        run(() => {
-          store.push({
-            data: {
-              type: BOOK_CLASS_PATH,
-              id: UNFETCHED_PROJECTION_ID,
-              attributes: {
-                title: 'Carry On! Mr. Bowditch',
-              },
-            },
-          });
-          store.preloadData({
-            data: {
-              type: BOOK_CLASS_PATH,
-              id: FETCHED_PROJECTION_ID,
-              attributes: {
-                title: `Mr. Popper's Penguins`,
-              },
+      run(() => {
```

Comment:
it seems this is mostly whitespace changes, confirm?

Reply:
Yes, not sure why switching from `skip` to `test` suddenly changes everything.